packages/build/tests/telemetry/tests.js

```diff
@@ -109,6 +109,18 @@ test('Telemetry reports build success', async (t) => {
   t.snapshot(snapshot)
 })
 
+test('Telemetry reports local plugins success', async (t) => {
+  const { telemetryRequests } = await runWithApiMock(t, 'plugin_success')
+  const snapshot = telemetryRequests.map(normalizeSnapshot)
+  t.snapshot(snapshot)
+})
+
+test('Telemetry reports package.json plugins success', async (t) => {
```

Comment:
Wondering if we would also want to add a test for plugins with `installType`=`netlify.toml` or `ui`?

Reply:
Just added :+1: